src/lib/CollectStore.js

```diff
@@ -106,7 +106,7 @@ export default class CollectStore {
   }
 
   updateRunningJob (job) {
-    if (job.state !== jobs.JOB_STATE.RUNNING) {
+    if (job.state === jobs.JOB_STATE.DONE || job.state === jobs.JOB_STATE.ERRORED) {
```

Comment:
That exactly what we need here but I think we should rename the whole part dealing with _running_ jobs, as we now handle both _running_ and _queued_.

I suggest `runningOrQueuedJobs` or `uncompletedJobs`. What do you think ?

Reply:
Agree, but complete could mean 'success' in some cases. What about `updateUnfinishedJob` ?

Reply:
Renamed to `updateUnfinishedJob` 👍

Reply:
Thanks, and we have also a property named `runningJobs` [in the CollectStore](https://github.com/cozy/cozy-collect/blob/master/src/lib/CollectStore.js#L38), may you also update it ? Thanks !

Reply:
Sure, I fixed up in the last commit :)

Reply:
Oops, sorry I missed that ! 🙏